Update notebooks (#148)
MiBiPreT/mibitrans: docs/implementation/implemenation.md

> # Implemenation

Reviewer: ```suggestion
# Implementation
```

Author: Changed it in document, file title and mkdocs.yml

## Architecture

The code is structured in a modular fashion to ensure easy extension of functionality by the (future) development team as well as outside collaborators. To stimulate embedding in the scientific community, users are encouraged to report bugs and suggest extensions and issues on the GitHub issue tracker, or even self-implemented features, through pull requests.

> <img src="mibitrans_schematics_structure.png" alt="Overview of `mibitrans` package structure and functionalities." width="500">

Reviewer: For me this image does not show in the local build. I would also suggest to stick to the markdown image format if you can. That said, the mkdocs asset manager is a bit of a pain. I got it working in the end by putting the image in a subdirectory of the implementation directory and using the following image link:

```suggestion
![Overview of `mibitrans` package structure and functionalities.](assets/mibitrans_schematics_structure.png)
```

Author: I added it to the general assets folder instead and it worked, at least when locally serving the documentation. Is that something that will not work once pushed to GitHub?

In the section **examples**, various example workflows, including application examples, provide inside into the application of `mibitrans`-functionalityin in the form of commented Jupyter-Notebooks.

## Research Software Management

We wish to reach with `mibitrans` a wide range of academic researchers with their research and education as well as societal stakeholders, like remediation consultants, to quickly evaluate contaminated field sites. To achieve that we follow in the development of `mibitrans`, the three aspects:

* scientific usefulness
* FAIR principles for Research Software
* software quality

`mibitrans` is written in Python, to align with the predominant trend of using Python for geoscientific modeling and data analysis. To further instigate adoption by the scientific community and other stakeholders, the development team follows the latest standards for the development and management of research software.

The open-source software package is being developed on a public GitHub repository, licensed under the permissive Apache 2.0 license. It applies version control, with released versions published on Zenodo and the Python Package Index (PyPI). 

> This last publication type enables a single-line installation with the \texttt{pip} tool, resulting in a very low usage threshold. 

Reviewer: ```suggestion
This last publication type enables a single-line installation with the `pip` tool, resulting in a very low usage threshold. 
```
